packages/ra-ui-materialui/src/button/CreateButton.stories.tsx

```diff
@@ -50,6 +50,7 @@ const AccessControlAdmin = ({ queryClient }: { queryClient: QueryClient }) => {
     const [resourcesAccesses, setResourcesAccesses] = React.useState({
         'books.list': true,
         'books.create': false,
+        'books.delete': false,
```

Comment:
I don't get why we need this, is it a fix ?

Reply:
It's a fix to avoid error messages in the tests output. Just a cleanup really